src/gui/qgsfilewidget.cpp

```diff
@@ -354,56 +341,61 @@ void QgsFileWidget::openFileDialog()
     return;
 
   if ( mStorageMode != GetMultipleFiles )
+    fileNames << fileName;
```

Comment:
why don't we need this anymore?
`QDir::toNativeSeparators( QDir::cleanPath( QFileInfo( fileName ).absoluteFilePath() ) );`

Reply:
We still do. It's done 3 lines later, in the for loop for every filename in `fileNames`.

In the rest of the code, I only deal with fileNames and no longer fileName.